### DISCUSSION TITLE
Fender Position and Angles

### DISCUSSION THREAD

**Discussion by IanSzalai** [FRCTeam3255/Robot2022-V#125] [General]
We only need two, since if we always assume we're on one side it will all work out.

Firstly I'll list the angles. The angle of the center line is 66° (±3°) and the fenders are all offset 45° from the center. That means the left (perspective from drivers station) is -21° (339°) and the right is 69°. Depending on how these are being used the sign might be flipped and they might be ±180°.

The positions are absolute. Importantly, this is going to be blue alliance, or left side. This is because all the autos paths are created on the blue side (and team color). These values assume 36x36 frame (bumpers) and being press up against fender. All values in meters. Left fender (6.98, 4.61), right fender (7.78, 2.84)

TL;DR:
Left fender: (6.98, 4.61), -21°
Right fender: (7.78, 2.84), 69°
